### PR TITLE
PDP 1383: Enable logging on DP Log Server

### PR DESCRIPTION
**Pull request by sasahoo-tibco**

### CODE REVIEW

File: charts/artifactmanager/values.yaml [hunk 1]
```diff
@@ -235,3 +235,7 @@ volumes:
 
     # -- When `persistentVolumeClaim.create` is `false`, then use this value to define an already-existing PVC.
     existingClaim: ""
+fluentbit:
+  image:
+    registry: cr.fluentbit.io/fluent/fluent-bit
```
Comment (pagarwal-tibco) [inline charts/artifactmanager/values.yaml:240]: we should not be using public docker images

**Review by pagarwal-tibco — COMMENTED** [review]
Is fluentbit config already present?

File: charts/artifactmanager/values.yaml (hunk 1 above)
Comment (sasahoo-tibco) [reply charts/artifactmanager/values.yaml:240]: This is not used.

**Comment by sasahoo-tibco** [comment]
> Is fluentbit config already present?

It was copied before. I updated this file in other branch.

File: charts/artifactmanager/values.yaml (hunk 1 above)
Comment (pagarwal-tibco) [reply charts/artifactmanager/values.yaml:240]: lets delete it then

**Review by pagarwal-tibco — APPROVED** [review]

**Review by pagarwal-tibco — APPROVED** [review]

**Pull request merged**